src/components/apptCard/ApptCard.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export const ApptCard = () => {
```

Comment:
nit: Spell out `AppointmentCard`? Your IDE should autocomplete to save you the extra keystrokes :stuck_out_tongue_closed_eyes:

Reply:
Done.